compiler/ast/src/accesses/static_access.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
```

Comment:
It looks like this PR re-introduces this file, which is currently not in the testnet3 branch. Is that unintentional?

Reply:
Oops it was not, I must have missed it during merging.